repository/src/main/java/org/apache/atlas/discovery/EntityDiscoveryService.java

```diff
@@ -1151,7 +1152,7 @@ private String getIndexName(IndexSearchParams params) throws AtlasBaseException
         if (StringUtils.isNotEmpty(aliasName)) {
             if(params.isAccessControlExclusive()) {
                 accessControlExclusiveDsl(params, aliasName);
-                aliasName = aliasName+","+VERTEX_INDEX_NAME;
```

Comment:
Make sure you have test case to test this flow.

Reply:
Yes